aws-k8s-cni.yaml

```diff
@@ -55,6 +58,20 @@ subpackages:
           ln -sf /usr/bin/egress-cni ${{targets.subpkgdir}}/app/egress-cni
           ln -sf /usr/bin/grpc-health-probe ${{targets.subpkgdir}}/app/grpc-health-probe
 
+  - name: "${{package.name}}-init-compat"
+    description: "Compatibility package to place binaries in the location expected by upstream helm charts"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/init
+          ln -sf /usr/bin/aws-vpc-cni-init ${{targets.subpkgdir}}/init/aws-vpc-cni-init
+          # Copy container networking plugins
+          binaries="bandwidth bridge dhcp dummy firewall host-device host-local ipvlan loopback macvlan portmap ptp sbr static tap tuning vlan vrf"
+          for bin in $binaries; do
+            cp /usr/bin/$bin ${{targets.subpkgdir}}/init/$bin
```

Comment:
Instead of copying these from the cni-plugins package, can we make this a subpackage in cni-plugins for this that copies the binaries to this location so that there isn't drift? For those following, we can't use a link here, because these will be copied to the host

Reply:
updated it!